shims/spark300db/src/main/scala/com/nvidia/spark/rapids/shims/spark300db/Spark300dbShims.scala

```diff
@@ -94,6 +94,8 @@ class Spark300dbShims extends Spark300Shims {
               wrapped.requiredSchema,
               wrapped.partitionFilters,
               wrapped.optionalBucketSet,
+              // TODO: Does Databricks have coalesced bucketing implemented?
```

Comment:
we probably need to go look at the source code for the various runtimes, if we have a jira we may be able to look at the release notes for each one as well. added a comment to https://github.com/NVIDIA/spark-rapids/issues/502